docs/src/pages/index.tsx

```diff
@@ -12,17 +27,154 @@ export default function Home() {
         css={css`
           /* so that the rest of the content has a higher z-order than the image */
           position: relative;
+          width: 100%;
+          max-width: 80rem;
+          margin: 0 auto;
+
+          > * + * {
+            margin-top: 5.25rem;
+          }
         `}
       >
-        <Heading type="display">
+        <div>
+          <Heading type="display">
+            <div
+              css={css`
+                max-width: 20ch;
+                font-size: 3rem;
+                line-height: 1.35;
+              `}
+            >
+              Open source design system for your next travel project.
+            </div>
+          </Heading>
+
+          <div
+            css={css`
+              margin-top: 3rem;
+            `}
+          >
+            <Inline spacing="small">
+              <ButtonLink
+                size="large"
+                as={Link}
+                iconRight={<ArrowRight />}
+                to="/getting-started/for-designers/kiwi"
```

Comment:
It was my understanding that the paths would be similar to how they are now. That would mean that the first tab would not have a separate path. So this path would be `/getting-started/for-designers/` (without `kiwi` -- like: https://orbit.kiwi/getting-started/for-designers/).

Is that not the plan, @silvenon ? It applies many other places (e.g., with "guidelines" below), so it would help for linking in general to know the rule to follow.

Reply:
Will fix this in separate PR, thanks!